NitroxServer/Server.cs

```diff
@@ -72,7 +74,27 @@ public void Save()
             }
 
             IsSaving = true;
-            worldPersistence.Save(world, serverConfig.SaveName);
+
+            if (worldPersistence.Save(world, serverConfig.SaveName) && !string.IsNullOrWhiteSpace(serverConfig.PostSaveCommandPath))
```

Comment:
```suggestion
            bool savedSuccessfully = worldPersistence.Save(world, serverConfig.SaveName);
            if (savedSuccessfully && !string.IsNullOrWhiteSpace(serverConfig.PostSaveCommandPath))
```
Seeing [sunrunners opinion](https://github.com/SubnauticaNitrox/Nitrox/pull/1519#discussion_r667355638) on logic inside ifs, this is probably fitting better as calling `worldPersistence.Save()` has not really something to do with checking if the post save command can be executed.